tests/unit/test_sharepoint.py

```diff
@@ -1,7 +1,20 @@
+import os
+
 from viadot.sources import Sharepoint
 
 
 def test_to_df(sharepoint_url):
     s = Sharepoint(config_key="sharepoint_prod")
     df = s.to_df(sharepoint_url)
     assert not df.empty
+
+
+def test_download_file(sharepoint_url):
+    s = Sharepoint(config_key="sharepoint_prod")
```

Comment:
Let's find a way not to publicly declare we don't have a test environment. 😅 If there is actually no dev or test envs, maybe we can create a fixture for `Sharepoint` and then this config key can be taken from an env variable that we set just for testing, eg. `VIADOT_TEST_SHAREPOINT_CONFIG_KEY`?

Reply:
[✨ Added support for sharepoint_config_key](https://github.com/dyvenia/viadot/pull/608/commits/0d21f22d013bd32ad4801aabf6051cdf0a8ad7b3)